data-client/src/main/java/org/wikipedia/wikidata/DataValue.kt

```diff
@@ -87,7 +87,7 @@ sealed class DataValue(val type: String) {
     //        "language": "ko"
     //    }
     //    }
-    class MonoLingualText_partial() : DataValue(TYPE) {
+    class MonoLingualText_partial(val value: WikiBaseMonolingualTextValue) : DataValue(TYPE) {
```

Comment:
If we have achieved full mapping of this data value then we can get rid of the `_partial` suffix, my intention with them was "I have no idea if this is absolutely the true network model"

Reply:
Updated.